### ISSUE TITLE
Add text indicating a transcript is available

### ISSUE THREAD

**Issue by Supravisor** [freeCodeCamp/freeCodeCamp#67148]
### Describe the Issue

In the A2 and B1 English for Developers certificates, some challenges include an audio component. The instruction text does not indicate a transcript is also available.

### Affected Page

https://www.freecodecamp.org/learn/b1-english-for-developers/learn-how-to-use-adjectives-in-conversations/dialogue-2-rehearsing-pitch

### Your code

```



```

### Expected behavior

Add a note:

> For accessibility, transcripts are available for learners with hearing impairments; we recommend using them only if needed.

### System

- Device: [e.g. iPhone 6, Laptop]
- OS: [e.g. iOS 14, Windows 10, Ubuntu 20.04]
- Browser: [e.g. Chrome, Safari]
- Version: [e.g. 22]

**Comment by huyenltnguyen** [comment]
Thank you for opening the issue.

We haven't received reports of users being unable to find transcripts or needing clearer indication of their availability.

Even if we do need to improve how transcripts are surfaced, adding more text is not the right solution as it effectively offloads the work to users (making them read more) rather than improving the UI itself. And the change itself needs to be made to almost all language challenges (not just English but Spanish and Chinese as well).

See https://github.com/freeCodeCamp/freeCodeCamp/issues/66953#issuecomment-4264260815.

We'll have major updates to language curricula in the future, so we are not going to proceed with this change.

**Comment by Supravisor** [comment]
> We haven't received reports of users being unable to find transcripts or needing clearer indication of their availability.

Sight challenged Campers will not know a transcript exists like the one on Discord:
> Originally the lectures did not mention the transcript, then a visually challenged Camper raised it on Discord, then the note was added.
> https://github.com/freeCodeCamp/freeCodeCamp/pull/61597/changes#diff-47bfae57479c78470f885850a31990a25c3f8ee1d077a2f9d6d3d9cc3d2b5808L11-L13
> [#58762](https://github.com/freeCodeCamp/freeCodeCamp/issues/58762)

At that time there were no other reports of "users being unable to find transcripts or needing clearer indication of their availability".

The A2 and B1 English for Developers certificates recently added a note for the quizzes. #66953

The Full Stack Developer Certificate before it was split into separate certificates also included a note for the lectures.

Not including a note would be inconsistent with those approaches.

Also, accessibility issues are a barrier to learning.

> And the change itself needs to be made to almost all language challenges (not just English but Spanish and Chinese as well).

I am willing to create meta issues for each language, and sub-issues as required.
If this issue is reopened, I can open English as a meta issue, then add A1 and B1 English for Developers certificates and any others as sub-issues, then create PRs.
